bioblend/galaxy/config/__init__.py

```diff
@@ -72,3 +72,13 @@ def whoami(self) -> dict:
         """
         url = self.gi.url + "/whoami"
         return self._get(url=url)
+
+    def reload_toolbox(self) -> None:
+        """
+        Reload the Galaxy toolbox (but not individual tools)
+
+        :rtype: None
+        :return: None
+        """
+        url = self.gi.url + "/toolbox"
```

Comment:
I think this should be:
```suggestion
        url = f"{self._make_url()}/toolbox"
```
?